errorreporting/pom.xml

```diff
@@ -0,0 +1,54 @@
+<?xml version='1.0' encoding='UTF-8'?>
+<project xmlns="http://maven.apache.org/POM/4.0.0" xmlns:xsi="http://www.w3.org/2001/XMLSchema-instance" xsi:schemaLocation="http://maven.apache.org/POM/4.0.0 http://maven.apache.org/xsd/maven-4.0.0.xsd">
+  <modelVersion>4.0.0</modelVersion>
+  <groupId>com.google.cloud</groupId>
+  <artifactId>java-docs-samples-errorreporting</artifactId>
+  <version>0.0.1-SNAPSHOT</version><!-- This artifact should not be released -->
+  <packaging>pom</packaging>
+  <name>Google Cloud Error Reporting Samples Parent</name>
+  <url>https://github.com/GoogleCloudPlatform/java-docs-samples/errorreporting</url>
```

Comment:
The url needs the `/tree/main/` part for it to work!

Reply:
Thanx!